controllers/news.controller.js

```diff
@@ -106,3 +105,17 @@ exports.postNewComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getAllUsers = (req, res) => {
+  const table = "users";
+  receiveAll(table).then((data) => {
+    data.forEach(item =>{
+      if(!item.username || !item.name || !item.avatar_url){
+        return Promise.reject({code: 404, msg: 'not found'})
+      }
```

Comment:
As your database contains columns of all these values, the keys should always exist on the data. I don't think you need to add this check in here, as you're checking in the test suite.

This check would also mean if someone inserted a user without an ID, your endpoint would throw an error, which may not be the functionality you want.